app/views/setup/welcome.html.slim

```diff
@@ -11,7 +11,7 @@ h1 Initial CaaS Platform Configuration
         .input-group
           = f.text_field :dashboard, value: @dashboard, class: "form-control", required: true
           span class="input-group-addon"
-            a data-toggle="tooltip" data-placement="left" title="Hostname or IP of the node running this web interface."
+            a data-toggle="tooltip" data-placement="left" title="Fully qualified domain name used to reach Velum within the internal network. The external fqdn will have to be provided in a later step."
```

Comment:
I think it's important to somehow state that an IP address is also allowed.

Reply:
updated